src/redux/fetch.ts

```diff
@@ -0,0 +1,20 @@
+export enum Method {
+  'Put' = 'PUT',
```

Comment:
I think you can just leave key without `''`, just
```suggestion
  Put = 'PUT',
```